src/state/slices/addressBookSlice/addressBookSlice.ts

```diff
@@ -0,0 +1,62 @@
+import type { PayloadAction } from '@reduxjs/toolkit'
+import { createSlice } from '@reduxjs/toolkit'
+import type { ChainId } from '@shapeshiftoss/caip'
+
+export type AddressBookEntry = {
+  id: string
+  name: string
+  address: string
+  chainId: ChainId
+  createdAt: number
```

Comment:
q: I get the copypastaroo here but do we need to know when an address was added? Feels fluff.